elastic/apm-server
# ci(benchmarks): remove verbose output and post is inferred

## Motivation/summary

Remove verbose output to avoid exposing any secrets and fix deprecations.

```
2024-12-29T20:44:34.3427207Z Note: Unnecessary use of -X or --request, POST is already inferred.
2024-12-29T20:44:34.3430063Z   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
2024-12-29T20:44:34.3432180Z                                  Dload  Upload   Total   Spent    Left  Speed
2024-12-29T20:44:34.3432809Z 
2024-12-29T20:44:34.3995127Z   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 35.193.143.25:9243...
...

2024-12-21T20:35:07.3985574Z > POST /api/reporting/generate/pngV2 HTTP/2
2024-12-21T20:35:07.3986039Z > Host: 879181ea7d994bcd9335be2deead0283.us-central1.gcp.cloud.es.io:9243
2024-12-21T20:35:07.3986543Z > authorization: Basic YW*********
2024-12-21T20:35:07.3986863Z > user-agent: curl/7.81.0
2024-12-21T20:35:07.3987141Z > accept: */*
2024-12-21T20:35:07.3987386Z > kbn-xsrf: true
2024-12-21T20:35:07.3987643Z > content-length: 440
2024-12-21T20:35:07.3987970Z > content-type: application/x-www-form-urlencoded
```

at https://github.com/elastic/apm-server/actions/runs/12446967846/job/34749950822

## Checklist




For functional changes, consider:
- Is it observable through the addition of either **logging** or **metrics**?
- Is its use being published in **telemetry** to enable product improvement?
- Have system tests been added to avoid regression?

## How to test these changes

In the CI, see this [build](https://github.com/elastic/apm-server/actions/runs/12547499384)

## Related issues
